indico/modules/events/ical.py

```diff
@@ -145,6 +146,16 @@ def generate_event_component(
     if data['description']:
         component['description'] = data['description']
 
+    # If the user exists and the add_icloud_alerts preference isn't blank
+    if user and user.settings.get('add_ical_alerts', False):
```

Comment:
```suggestion
    if user and user.settings.get('add_ical_alerts'):
```

the default is coming from the SettingsProxy definition if omitted